src/renderer/components/dialogs/QrCode.tsx

```diff
@@ -293,12 +294,12 @@ export function QrCodeScanQrInner(
       </DeltaDialogBody>
       <DeltaDialogFooter>
         <DeltaDialogFooterActions style={{ justifyContent: 'space-between' }}>
-          <p className={'delta-button bold primary'} onClick={openMenu}>
+          <Button type='primary' onClick={openMenu}>
```

Comment:
For buttons inside `DialogFooter` (former `DeltaDialogFooter`) I've introduced the `FooterActionButton` (https://github.com/deltachat/deltachat-desktop/blob/master/src/renderer/components/Dialog/FooterActionButton.tsx) component which looks very similar to this one here.

Currently _all_ buttons in dialogues are _always_ `bold` and `primary`, so no need to be highly configurable in this context.

When rebasing this PR to `master` (with my refactorings) I'd suggest to use your new `Button` inside of `FooterActionButton` where `bold` and `primary` is always set (composition always preferable in React).

Reply:
Yeah agreed. But your `FooterActionButton` must use my Button component IMO.